### PR TITLE
Mesh.mesh() != Mesh.mesh()

### PR DESCRIPTION
**Pull request by Jotschi**

### CODE REVIEW

**Comment by philippguertler** [comment]
How does logging work? How do I know from which instance a log message originates from?

**Comment by philippguertler** [comment]
Delete ImageManipulatorService

**Comment by philippguertler** [comment]
Write lock handling in HandlerUtilities is still static

**Comment by philippguertler** [comment]
RestPluginRegistry#apiNameSyncSet is static

**Comment by philippguertler** [comment]
Disable embedded ES in multi mesh mode

**Comment by philippguertler** [comment]
PermissionStore is still static

File: services/keycloak-auth/src/main/java/com/gentics/mesh/auth/MeshOAuth2ServiceImpl.java [hunk 1]
```diff
@@ -72,10 +73,14 @@
 	protected Database db;
 	protected BootstrapInitializer boot;
 
+	private final Provider<EventQueueBatch> batchProvider;
```
[the remark this answers is not in the document]
Comment (Jotschi) [reply services/keycloak-auth/src/main/java/com/gentics/mesh/auth/MeshOAuth2ServiceImpl.java]: Unclear

**Comment by Jotschi** [comment]
> How does logging work? How do I know from which instance a log message originates from?
> 
> Possible solution is prefixing the thread pools so that each thread can be associated with each instance. The thread name can then be part of the log message

I will rework the logging. In Vert.x 4 the Vert.x Logger Factories will be deprecated. I'll inject the loggers for each instance. This will be done in a dedicated PR.

**Pull request closed without merging**